vue-frontend/src/components/android-app-development/SuccessStorySection.vue

```diff
@@ -97,39 +98,40 @@
               }"
               :loop="true"
               :loopedSlides="50"
-              class="swiper-container tw-h-[600px] lg:tw-h-[700px] tw-px-2"
+              class="swiper-container tw-h-[600px] tw-px-2"
               @swiper="setSwiperRef"
               @slideChange="onSlideChange"
             >
               <swiper-slide v-for="(client, index) in clients" :key="index">
                 <div
-                  class="tw-my-10"
+                  class="tw-my-10 tw-flex tw-flex-col tw-items-center tw-justify-center"
                   :class="
                     activeIndex == index || activeindex == 0
-                      ? 'tw-grayscale-0 tw-animate-slideLeft'
-                      : 'tw-grayscale'
+                      ? 'tw-animate-zoomEffect'
+                      : ''
```

Comment:
client review doesn't look center, I mean the top and bottom margins don't look the same.